octopoes/tests/integration/test_unicode.py

```diff
@@ -0,0 +1,85 @@
+import os
+import time
+import uuid
+from datetime import datetime
+
+import pytest
+
+from octopoes.api.models import Declaration, Observation
+from octopoes.connector.octopoes import OctopoesAPIConnector
+from octopoes.models import DeclaredScanProfile, ScanLevel
+from octopoes.models.ooi.dns.zone import Hostname
+from octopoes.models.ooi.network import Network
+from octopoes.models.origin import OriginType
+
+if os.environ.get("CI") != "1":
+    pytest.skip("Needs XTDB multinode container.", allow_module_level=True)
+
+USE_UNICODE = True
+NAMES = ["🐱", "★.com", "🐈"] if USE_UNICODE else ["cat", "xn--p3h.com", "boefje"]
```

Comment:
I don't see the purpose of `USE_UNICODE` here.

Reply:
Perhaps parametrizing this test would be better (testing with and without encoding)

Reply:
The purpose was for a developer to be able to quickly switch ascii, unicode (for the purpose for which the test was built). I think it does not hurt keeping but if we insist on removing it, I am fine with that.
As for doing both tests with and without, we already have coverage for that in the other tests specifically https://github.com/minvws/nl-kat-coordination/blob/d11aac0d539a5a6101906951cce81331bf0ee9fe/octopoes/tests/integration/test_api_connector.py#L1-L208

Reply:
I think it is just cruft that was useful when debugging but it not going to be used again. And if you want to change the test, you can just edit the NAMES list if you are editing the file, it's not that the `USE_UNICODE` makes that much easier.

Reply:
Addressed in faf8fd1adffdac0d417bc66db849623ba2e241ce.